### ISSUE TITLE
It should contain Quarkus builder

### ISSUE THREAD

**Issue by cancinar** [paketo-buildpacks/java#1503]
## Describe the Enhancement
Java Builder should contain Quarkus Builder as Spring Boot. 

## Possible Solution
Adding Quarkus builder 

```
  [[order.group]]
    id = "paketo-buildpacks/quarkus"
    optional = true
    version = "0.4.0"
```

## Motivation
We are using Jammy Base Builder which uses Java Builder under the hood. However, we are unable to build Quarkus applications.

**Comment by dmikusa** [comment]
tl;dr - 100%, we want to add this to the builder.

-----

Long story: There is a layer limit on container images, it's part of the Linux kernel itself so it's not something we can just adjust, but this is currently preventing us from adding more buildpacks to the Java buildpack.

We have some plans for fixing that, to squash layers on the images, but it hasn't been implemented so until that happens we cannot add more buildpacks.

We are also working on a Java-specific builder that would have a wider range of buildpacks but only buildpacks around building Java apps. See [here](https://github.com/paketo-community/builder-noble-java-tiny), but that's early stages and we haven't expanded the order groups for it yet.

You can presently build apps using the Quarkus buildpack, but you need to manually configure a few more things, check out this [sample app](https://github.com/paketo-buildpacks/samples/tree/main/java/native-image/quarkus-native-image-maven), if you haven't seen it yet.
